components/service/glean/docs/pings/pings.md

```diff
@@ -2,15 +2,27 @@
 
 If data collection is enabled, glean provides a set of pings that are assembled out of the box
 without any developer intervention.
-Every glean ping is in JSON format and contains a [common section](#the-ping_info-section)
-with shared information data included under the `ping_info` key.
-The following is a list of the pings along with the conditions that triggers them.
+Every glean ping is in JSON format and contains one or more of the [common sections](#ping-sections)
+with shared information data.
+The following is a list of these built-in pings:
 
 - [`baseline` ping](baseline.md)
 - [`events` ping](events.md)
 - [`metrics` ping](metrics.md)
 
-## The `ping_info` section
+## Ping sections
+
+There are two standard sections that are added to most pings, in addition to their
+core metrics and events content.
+
+- The [`ping_info` section](#The-ping_info-section) contains core metadata that
+  is included in **every** ping.
+- The [`client_info` section](#The-client_info-section) contains information
+  that identifies the client. It is included in most pings (including all
+  built-in pings), but may be excluded from pings where we don't want to connect
+  client information with the other metrics in the ping.
+
+### The `ping_info` section
```

Comment:
I see one additional section that's common in all 3 pings (baseline, event, metrics) which is the `metrics` section. Does that need to be elaborated on here too, similar to the `ping_info` and `client_info` sections?

Reply:
`metrics` and `events` contain the actual telemetry.  `ping_info` and `client_info` are basically "metadata".  The `metrics` and `events` sections are covered in the `metrics` directory of docs -- but let's link to them from this sentence:

```
There are two standard sections that are added to most pings, in addition to their
core metrics and events content.
```